web/server.js

```diff
@@ -14,8 +15,12 @@ module.exports = function(config) {
       coinbase_secret: config.coinbase_secret
     }
   });
+  server.connection({
+    host: config.host,
+    port: config.port
+  });
 
-  server.pack.register(require("hapi-auth-bearer-token"), function(err) {});
+  server.register(require("hapi-auth-bearer-token"), function(err) {});
```

Comment:
I see. I guess we only really need to worry if the register API changes and breaks...